streamreplication/dynamo/stream.go

```diff
@@ -0,0 +1,249 @@
+package dynamo
+
+import (
+	"sync"
+	"time"
+
+	"github.com/aws/aws-sdk-go/service/dynamodbstreams"
+	ds "github.com/aws/aws-sdk-go/service/dynamodbstreams"
+	"github.com/cloudspannerecosystem/dynamodb-adapter/pkg/logger"
+)
+
+var (
+	inProcessShards      map[string]*dynamodbstreams.Shard
```

Comment:
Need to move these variable to struct fields as multiple streamers can be initiated at the same time.